Shuttle-back/src/main/java/com/shuttle/common/CollectionDTO.java

```diff
@@ -0,0 +1,22 @@
+package com.shuttle.common;
+
+import java.util.Collection;
+
+public class CollectionDTO<T> {
+	private long totalCount;
+	private Collection<T> results;
+	public long getTotalCount() {
```

Comment:
I don't think we need getters and setters for DTOs since they just get mapped 1-1 to JSON objects. Marking the properties as `public` is enough.
Instead, consider writing a constructor that converts an `Obj` into an `ObjDTO`.

Reply:
I agree, will do